config/crd/bases/infra.contrib.fluxcd.io_terraforms.yaml

```diff
@@ -1,10 +1,9 @@
-
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.9.0
```

Comment:
could you downgrade `kubebuilder` to v0.7.x and re-run `make generate` and `make manifests` please?

Reply:
sure